addons/skin.estuary/xml/Variables.xml

```diff
@@ -376,8 +377,8 @@
 	<variable name="VideoInfoMainLabelVar">
 		<value condition="String.IsEqual(ListItem.DBType,set)">$INFO[ListItem.Title]</value>
 		<value condition="!String.IsEmpty(ListItem.TVShowTitle)">$INFO[ListItem.TVShowTitle]$INFO[ListItem.Year, ([COLOR grey],[/COLOR])]</value>
-		<value condition="!String.IsEmpty(ListItem.Title)">$INFO[ListItem.Title]$INFO[ListItem.Year, ([COLOR grey],[/COLOR])]</value>
-		<value>$INFO[ListItem.Label]$INFO[ListItem.Year, ([COLOR grey],[/COLOR])]</value>
+		<value condition="!String.IsEmpty(ListItem.Title)">$INFO[ListItem.Title]$INFO[ListItem.Year, ([COLOR grey],[/COLOR])]$INFO[ListItem.VideoVersionName, ([COLOR grey],[/COLOR])]</value>
```

Comment:
Same as above.

Reply:
Same answer. Will not work (and here it never would have).

Reply:
And in the video info dialog you want to see the version name unconditionally, btw. The dialog now displays art for a certain version (in future we want to have also different other meta data for the different versions - plot etc.). Something not possible with the current data model, though.